src/styles/components/_tile.scss

```diff
@@ -101,19 +111,18 @@
 
 .tile__hit-count {
   @include tile-hit-count-text;
-  float: right;
+  position: relative;
+  order: 1;
+  top: -32px;
+  right: -70px;
```

Comment:
```suggestion
@include tile-hit-count-text;
  position: absolute;
  order: 1;
  top: 0px;
  right: 20px;
```

Reply:
The tile hit count positioning is a little off. I think this fixes it?

Reply:
The tile hit count button positioning is off. This doesn't entirely fix it - on hover it's still wonky

Reply:
Yeah this is a function of the border width change on hover state. My googling has not yet revealed a solution for this, although I am sure there is one somewhere!

Reply:
Happy to wait and deal with this later if we need to .